src/planwise/client/components/nav.cljs

```diff
@@ -3,12 +3,17 @@
 
 ;; Navigation components
 
-(defn- li-menu-item [{:keys [selected item href title icon]}]
+(defn- li-menu-item [{:keys [selected item href title icon tab-number wizard-state]}]
   (let [is-selected? (or (= selected item)
                          (item selected))]
-    [:li {:class (when is-selected? "active")}
+    [:li {:class (str
+                  (when is-selected? "active ")
+                  (when wizard-state (str (name wizard-state) " wizard")))}
```

Comment:
Those styles can be applied to each of the classes yielded by the different wizard states. Eg.

```css
.visited, .unvisited, .visiting {
  /* styles from wizard */
}
```

You can make the selector more specific if you're worried that you can later have clashes. Eg. `ul > li.visited, ul > li.unvisited, ul > li.visiting`

**If** that's not enough, adding a `wizard` class to the `ul` and using the selector `ul.wizard > li` is better than adding the `wizard` class to each `li`.